.github/workflows/publish_release_vermeer_image.yml

```
@@ -0,0 +1,54 @@
+      branch:
+        required: true
+        default: ''
+        description: 'The branch name should be like *-x.x.x, for example release-1.0.0'
+      
+jobs:
+  build_latest:
+    runs-on: ubuntu-latest
+    env:
+      REPOSITORY_URL: apache/incubator-hugegraph-computer
+      BRANCH: ${{inputs.branch}}
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)